node/src/chain_spec.rs

```diff
@@ -680,26 +680,29 @@ fn centrifuge_genesis(
 }
 
 fn altair_genesis(
-	initial_authorities: Vec<(altair_runtime::AccountId, altair_runtime::AuraId)>,
-	mut endowed_accounts: Vec<altair_runtime::AccountId>,
+	initial_authorities: Vec<(AccountId, AuraId)>,
+	mut endowed_accounts: Vec<AccountId>,
 	endowed_evm_accounts: Vec<([u8; 20], Option<u64>)>,
-	total_issuance: Option<altair_runtime::Balance>,
+	total_issuance: Option<Balance>,
 	id: ParaId,
-	council_members: Vec<altair_runtime::AccountId>,
+	council_members: Vec<AccountId>,
 ) -> altair_runtime::GenesisConfig {
 	let chain_id: u32 = id.into();
 
 	endowed_accounts.extend(endowed_evm_accounts.into_iter().map(|(addr, id)| {
 		let chain_id = id.unwrap_or_else(|| chain_id.into());
-		AccountConverter::<centrifuge_runtime::Runtime, centrifuge_runtime::LocationToAccountId>::convert_evm_address(chain_id, addr)
+		AccountConverter::<
+			altair_runtime::Runtime,
+			altair_runtime::xcm::LocationToAccountId,
+		>::convert_evm_address(chain_id, addr)
```

Comment:
These were wrong. When adding a new node to the network, does the node initialize its configuration with this genesis?

Reply:
This section does not effect Altair but only any Altair-based chain spec from now on, e.g. Altair local or a theoretical Altair-based testnetwork which we might create in the future. 

This function alters storage and thus leads to a different genesis hash. Since we haven't used any converter for the genesis Altair, there is nothing to worry. Thanks for pointing this out and fixing though!

Reply:
Thanks for the explanation!